### DISCUSSION TITLE
Difference between "simple_outflow" and "open" boundary condition

### DISCUSSION THREAD

**Discussion by xh-ui** [epochpic/epoch#673] [Q&A]
Dear all,
What's the difference between "simple_outflow" and "open" boundary condition?  They both don't change the particle or laser or waves.

**Comment by TomGoffrey** [comment]
I'm fairly sure these are the same. After a quick look through the source code it looks like these are treated as synonyms. The documentation should be updated to reflect this.

If there is any doubt you could run some quick simulations to compare - I'd expect identical results.

**Reply by xh-ui** [reply]
Thank you!  I think I should suggest the creators of visit project.  @Status-Mirror
